script.js

```diff
@@ -1,12 +1,98 @@
 //You can edit ALL of the code here
+
+const rootElem = document.getElementById("root");
+const inputSearch = document.querySelector("#liveSearch");
+const pCounter = document.querySelector("#counter");
+const selectEpisodes = document.querySelector("#selectEpisodes");
+const selectShow = document.querySelector("#selectShows");
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  const allShows = getAllShows();
+  makeSelectShows(allShows); //level 400
+  fetchingShows();
+}
+
+function makeSelectShows(shows) {
+  shows.forEach((eachShow) => {
+    const option = document.createElement("option");
+    option.textContent = eachShow.name;
+    option.value = eachShow.id;
+    selectShow.appendChild(option);
+  });
+}
+
+function fetchingShows() {
+  selectShow.addEventListener("change", () => {
+    if (selectShow.value === "empty")
+      selectEpisodes.innerHTML = `<option value="all">All Episodes</option>`;
+    else {
+      fetch(`https://api.tvmaze.com/shows/${selectShow.value}/episodes`) //Level 350
+        .then((response) => response.json())
+        .then((episodes) => {
+          makeSelectEpisodes(episodes, episodes); //Level 300
```

Comment:
So do you have to pass two parameters to `makeSelectEpisodes` function, while both of them are same?